.github/workflows/_sonarcloud.yml

```
@@ -0,0 +1,90 @@
+name: Perform SonarCloud Scan
+
+on:
+  workflow_call: 
+
+    inputs:
+      sonar-config:
+        description: "Configuration for SonarCloud"
```

```suggestion
        description: "Configuration for Sonar"
```
I like just calling this "Sonar" as all their offering are under that name. Please change all of these.